bitnami/opencart/requirements.yaml

```diff
@@ -1,5 +1,8 @@
 dependencies:
-- name: mariadb
-  version: 7.x.x
-  repository: https://charts.bitnami.com/bitnami
-  condition: mariadb.enabled
+  - name: mariadb
+    version: 7.x.x
```

Comment:
Could we also take the advantage of this PR to upgrade mariadb to `8.x.x`?

Reply:
good point, done